torchrl/envs/transforms/transforms.py

```diff
@@ -453,10 +453,16 @@ def _step(self, tensordict: TensorDictBase) -> TensorDictBase:
 
         return tensordict_out
 
-    def set_seed(self, seed: int, static_seed: bool = False) -> int:
-        """Set the seeds of the environment."""
+    def set_seed(
+        self, seed: Optional[int] = None, static_seed: bool = False
+    ) -> Optional[int]:
+        # This method is not used in transformed environments
```

Comment:
Is this true? The message matches the one from _set_seed

Reply:
Maybe we could use the docstring above?

Another note: I don't remember why we're not using _set_seed to be honest. I guess it's because the seed iteration would occur twice if `transformed_env._set_seed` would call ˋbase_env.set_seed`, but what if `transformed_env._set_seed` would call ˋbase_env._set_seed`?

Reply:
> Is this true? The message matches the one from _set_seed

Oh sorry it was not meant to be present in `set_seed`

> Maybe we could use the docstring above?
> 
> Another note: I don't remember why we're not using _set_seed to be honest. I guess it's because the seed iteration would occur twice if `transformed_env._set_seed` would call ˋbase_env.set_seed`, but what if `transformed_env._set_seed` would call ˋbase_env._set_seed`?

Basically `_set_seed` is only called in the original implementation of `set_seed` in `BaseEnv`. Given that `TransformedEnv` and `BatchedEnv` override `set_seed` (this would not normally be the case for inheritors of `BaseEnv` and is the case just for these two examples as they are wrappers) the inner `_set_seed` is not called anymore and we just implement it with a pass (whatever we implement it with doesn't matter as it is never called)